src/brasil/gov/tiles/tiles/templates/mediacarousel.pt

```diff
@@ -52,12 +58,15 @@
                         </a>
                     </tal:items>
                 </div>
-                <div i18n:domain="brasil.gov.tiles" tal:condition="python:field=='uuids' and not(items)" i18n:translate="">
+                <div i18n:domain="brasil.gov.tiles" tal:condition="python:field['id']=='uuids' and not(items)" i18n:translate="">
                     Drag a folder or collection to populate the tile.
                 </div>
-                <div tal:condition="python:field=='footer_text' and collection" class="mediacarousel-footer-container">
-                    <a class="mediacarousel-footer-link" tal:content="view/data/footer_text"
-                       tal:attributes="href collection/absolute_url"></a>
+                <div tal:condition="python:field['id']=='footer_text' and view.data.get('footer_text')" class="mediacarousel-footer-container">
```

Comment:
here you are asking 3 times for `footer_text` it would be better to initialize a variable, even as this may be not costly in terms of performance.